docs/consume-packages/install-use-packages-nuget-cli.md

```diff
@@ -1,118 +1,123 @@
+```cli
+nuget install Newtonsoft.Json -OutputDirectory packages
+```
+
+Instead of specifying a package to install, you can specify an existing *packages.config* file in the current or another directory. The `install` command installs all the packages listed in the *packages.config* file.
 
-    ```cli
-    nuget install Newtonsoft.Json -OutputDirectory packages
-    ```
+```cli
+nuget install packages.config
+```
 
-Alternatively, you can use the following command to install a NuGet package using an existing `packages.config` file to the *packages* folder. This does not add the package to your project dependencies, but installs it locally.
+For example, the following command installs all the packages listed in *packages.config* in the *config* subdirectory to the *packages* subdirectory:
 
 ```cli
-nuget install packages.config -OutputDirectory packages
+nuget install config\packages.config -OutputDirectory packages
+
 ```
 
 ## Install a specific version of a package
 
-If the version is not specified when you use the [install](../reference/cli-reference/cli-ref-install.md) command, NuGet installs the latest version of the package. You can also install a specific version of a Nuget package:
+The `install` command installs the latest version of a package unless you specify a different version. To install a specific version of a package, use the `-Version` option:
 
 ```cli
 nuget install <packageID | configFilePath> -Version <version>
 ```
 
-For example, to add version 12.0.1 of the `Newtonsoft.json` package, use this command:
+For example, to install version 12.0.1 of the `Newtonsoft.json` package, use:
 
 ```cli
 nuget install Newtonsoft.Json -Version 12.0.1
 ```
 
-For more information on the limitations and behavior of `install`, see [Install a package](#install-a-package).
-
-## Remove a package
-
-To delete one or more packages, delete the packages you want to remove from the *packages* folder.
-
-If you want to reinstall packages, use the `restore` or `install` command.
-
 ## List packages
 
-You can display a list of packages from a given source using the [list](../reference/cli-reference/cli-ref-list.md) command. Use the `-Source` option to restrict the search.
+Use the [list](../reference/cli-reference/cli-ref-list.md) command to display a list of packages. Use the `-Source` option to restrict the list.
 
 ```cli
 nuget list -Source <source>
 ```
 
-For example, list packages in the *packages* folder.
+For example, to list packages in the *packages* subdirectory of *MyProject*, use:
 
 ```cli
-nuget list -Source C:\Users\username\source\repos\MyProject\packages
+nuget list -Source C:\Users\%USERNAME%\source\repos\MyProject\packages
 ```
 
-If you use a search term, the search includes names of packages, tags, and package descriptions.
+If you don't specify a source, `list` uses all sources defined in the *NuGet.Config* global configuration file. If *NuGet.Config* doesn't specify any sources, `list` uses a default feed of nuget.org.
```

Comment:
This information deviates from the context of the doc which is nuget.exe for installing packages.

Reply:
@nkolev92 - Not sure what you mean - this series of articles includes the 'list' capability.

Reply:
The part talking about sources being used from nuget.config. 

The `list` functionality lists the packages from a specific source. It just so happens that the packages folder itself has the shape of a source, so it can be used as an approximation to view the packages installed the cunrrent repo/projects.

Reply:
@nkolev92 - Right but if you start a new project with no dependencies and then do nuget list, it starts listing from nuget.org, is what I'm trying to say.

Reply:
Unfortunately the command itself is misnamed when compared to commands such as dotnet list package, https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-list-package, which is a true command that lists your installed packages (only works for SDK based projects).

This list command is more of a `search` command, where it searches for a package on certain sources. 
It's equivalent to the `Find-Package` command in PMC. 

The current usage is not the primary intended one for this command. 
I think we should tweak it to link to the command description and then explain this is a way of getting an approximation of installed packages on the commandline

Reply:
@nkolev92 - OK, I removed that sentence. (The first sentence already links to the command.)